ecoscope/io/earthranger.py

```diff
@@ -187,38 +114,25 @@ def get_subjects(
         """
         Parameters
         ----------
-        include_inactive
-            Include inactive subjects in list.
-        tracks_since
-            Include tracks since this timestamp
-        tracks_until
-            Include tracks up through this timestamp
-        bbox
-            Include subjects having track data within this bounding box defined by a 4-tuple of coordinates marking
+        include_inactive: Include inactive subjects in list.
+        tracks_since: Include tracks since this timestamp
+        tracks_until: Include tracks up through this timestamp
+        bbox: Include subjects having track data within this bounding box defined by a 4-tuple of coordinates marking
             west, south, east, north.
-        subject_group
-            Indicate a subject group for which Subjects should be listed.
-        name : UUID
-            Find subjects with the given name.
-        updated_since
-            Return Subject that have been updated since the given timestamp.
-        render_last_location
-            Indicate whether to render each subject's last location.
-        tracks
-            Indicate whether to render each subject's recent tracks.
-        id
-            A comma-delimited list of Subject IDs.
+        subject_group: Indicate a subject group for which Subjects should be listed.
+        name : Find subjects with the given name [UUID]
+        updated_since: Return Subject that have been updated since the given timestamp.
+        render_last_location: Indicate whether to render each subject's last location.
+        tracks: Indicate whether to render each subject's recent tracks.
```

Comment:
I think we should set this to false and hide from user as an param. I think this  param applies mainly to the web app.